tests/cmd-args.py

```diff
@@ -2,10 +2,15 @@
 
 from os import listdir
 from subprocess import DEVNULL, PIPE, Popen
+from shutil import which
 
 failed = False
+no_dot = which('dot') is None
 
-for f in filter(lambda x: x.startswith('llvm'), listdir()):
+for f in filter(lambda x: x.startswith('llvm') and not x.endswith('.cpp'), listdir()):
+    # skip wrapper scripts if 'dot' is missing as they depend on it
+    if no_dot and f in ('llvmdda-dump', 'llvmdg-show'):
+        continue
```

Comment:
Nice catch! It would be nicer to list these executables in the output with a `SKIP` result though (and maybe even with a reason for that), than to skip them silently.